PyRDF/CallableGenerator.py

```diff
@@ -96,7 +96,7 @@ def mapper(node_cpp, node_py=None):
                 # recursive call
```

Comment:
Now this part becomes a little fuzzy. While before we were actually returning PyROOT class instances in the form of lazy transformations and booked actions, now we can also return fundamental types like strings and vectors.